app/js/components/drugOrderEntry/index.jsx

```diff
@@ -24,6 +24,20 @@ export class SearchAndAddOrder extends React.Component {
     draftOrder: {},
   };
 
+  componentWillMount() {
```

Comment:
The `componentWillMount` lifeCycle method is marked as `legacy`, therefore, it is advised not to be used in new code.please use `componentDidMount` instead.

Reply:
done!